tests/integration/steps_double_sign.go

```diff
@@ -13,112 +14,204 @@ func stepsDoubleSign(consumerName, doubleSignChain, doubleSignValidator string)
 				// slash on provider
 				chainID("provi"): ChainState{
 					ValPowers: &map[validatorID]uint{
-						validatorID("alice"): 500,
-						validatorID("bob"):   500,
+						validatorID("alice"): 509,
+						validatorID("bob"):   495,
+						validatorID("carol"): 0, // from 495 to 0
+					},
+				},
+				chainID(consumer1): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 509,
+						validatorID("bob"):   495,
+						validatorID("carol"): 495, // not slashed on consumer1 yet
+					},
+				},
+				chainID(consumer2): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 509,
+						validatorID("bob"):   495,
+						validatorID("carol"): 495, // not slashed on consumer2 yet
+					},
+				},
+			},
+		},
+		{
+			// relay power change to consumer1
+			action: relayPacketsAction{
+				chain:   chainID("provi"),
+				port:    "provider",
+				channel: 0, // consumer1 channel
```

Comment:
Neat that you can relay by channel, makes the tests very clear

Reply:
Done.